src/commands/done.js

```diff
@@ -22,6 +23,18 @@ class DoneCommand extends Command {
       } else {
         this.log(chalk.yellow(CONSTANTS.DONE_INCORRECT_INDEX(args.INDEX)));
       }
+    } else if (args.INDEX === 'all') {
+      // Confirm
+      const prompt_msg = CONSTANTS.DONE_PROMPT_MSG;
+      let input = await cli.confirm(prompt_msg);      
```

Comment:
Maybe the `const` is not necessary

```suggestion
      let input = await cli.confirm(CONSTANTS.DONE_PROMPT_MSG);      
```

Reply:
Hi @brunocerecetto ! It's a detail